packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-node-ciemss.vue

```diff
@@ -135,7 +136,13 @@ watch(
 					method: 'dopri5',
 					inferred_parameters: id
 				},
-				engine: 'ciemss'
+				engine: 'ciemss',
+				metadata: {
+					workflowId: props.node.workflowId,
+					workflowName: useProjects().getAssetName(props.node.workflowId),
+					nodeId: props.node.id,
+					nodeName: props.node.displayName
+				}
```

Comment:
I am wondering if it will be better to have the metadata as a sibling rather than as a part of the simulation request. that is to say, rather than:

```
function makeForecastJobCiemss(simulationParam: SimulationRequest)
```

we have this as the functional signature

```
function makeForecastJobCiemss(simulationParam: SimulationRequest, metadata?: any)
```

This should make things a bit cleaner in my opinion:
- You don't need to add objects in, only to take it out later a second later because downstream has schema checks or whatnot
- Even if we can pass metadata down, we do not leak implementation detail to services that do not care about workflow-node-ids
- Less rippling effects due to typing changes

Reply:
Yeah I think that's ideal but I haven't find a simple enough way to include the metadata object as a separate request param with `RequestParam` annotation. So I included it in the request body object. I'll look a bit more to see if theres's a work around.

Reply:
Or we can just restructure the request body like following and use `@RequestBody` without additional request param
```
{
  metadata: any
  simulationRequest: SimulationRequest
}
```